Add a timestamp field to the contact form

clemente opened the pull request:
To debug double submissions, we're sending a timestamp and a random number in the contact form. The field Profession was suggested by them because it's unused.

Testing instructions.
1. https://pearson-test.sandbox.stage.opencraft.hosting/courses/course-v1:testing1+other2+2018_02/about and click „enquire now“
2. check in the HTML that the "oid" parameter has a nonsense value instead of their ID. Then you'll be able to submit the form without altering their DB. The form also has debug=1 so it will send you to a debug page
3. fill the form and submit it, it will go to a debug page
4. check the contents of the field 00N58000003di7f, it should have XXXXXXXX-YYYYY, where XXXXXXX is a timestamp (in ~~ms) and YYYYY a random number
5. Check that the code is in the right place (e.g. if some clicked the button twice, it would send different results

Reviewers: one of:
- @ciuin (if available)
- @bradenmacdonald 
- or TBD

clemente committed:
Add a timestamp field to the contact form

bradenmacdonald commented:
@clemente You can also just put this code on the live server or deploy a server using this branch as the theme, since it's only for debugging. We don't necessarily have to merge this.

Code looks good, but I think you should put the randomization _inside_ the `$("form#contact-form").submit(function (event){` event handler, or at least some way of finding out if that method is being called twice, such as putting something in there like `$("#00N58000003di7f").val($("#00N58000003di7f").val() + '-submit')` so we can tell if that handler is somehow getting called twice.

clemente commented:
@bradenmacdonald Thanks for catching this; I had it there and moved it for some reason.
Fixed (put inside), so that if someone sends the form without running the JS, the field will be empty.

Ok, deploying.
